bonsai_gym/gym_simulator3.py

```diff
@@ -106,6 +107,51 @@ def gym_simulate(self, gym_action):
         observation, reward, done, info = self._env.step(gym_action)
         return observation, reward, done, info
 
+    def run_random(
+        self, num_episodes: int = 10, num_iterations: int = 200, render: bool = False
+    ):
+
+        for episode in range(num_episodes):
+            terminal = False
+            iteration = 0
+            while not terminal:
+                iteration += 1
+                random_action = self._env.action_space.sample()
+                _, reward, terminal, _ = self._env.step(random_action)
+                log.info(
+                    f"Episode: #{episode}, iteration: #{iteration}, reward: {reward}, Terminal: {terminal}"
+                )
+                if iteration > num_iterations - 1:
+                    terminal = True
+                if not render or self._headless:
+                    if "human" in self._env.metadata["render.modes"]:
+                        self._env.render()
+
+    def run_exported_brain(
+        self,
+        num_episodes: int = 500,
+        num_iterations: int = 200,
+        render: bool = False,
+        exported_brain_url: str = "http://localhost:5000",
+        config: Union[Dict[str, Any], None] = None,
+    ):
+
+        prediction_endpoint = f"{exported_brain_url}/v1/prediction"
+        for _ in range(num_episodes):
+            self.episode_start(config=config)
+            terminal = False
+            iteration = 0
+            while not terminal:
+                iteration += 1
+                response = requests.get(prediction_endpoint, json=self.get_state())
+                action = response.json()
```

Comment:
does not end episode when terminal. I added this functionality in 
https://github.com/microsoft/bonsai-gym/pull/10